packages/squiggle-lang/src/rescript/Distributions/PointSetDist/PointSetDist_Scoring.res

```diff
@@ -14,3 +14,20 @@ module KLDivergence = {
       quot < 0.0 ? Error(Operation.ComplexNumberError) : Ok(-.answerElement *. logFn(quot))
     }
 }
+
+/*
+
+*/
+module LogScore = {
+  let logFn = Js.Math.log
+  let integrand = (priorElement: float, predictionElement: float, ~answer: float) => {
```

Comment:
maybe rename this to something, like, ``integrandAtXPoint``?

Reply:
after revamp comment is outdated.